tests/src/test/scala/common/rest/WskRestOperations.scala

```diff
@@ -1319,6 +1297,15 @@ trait RunRestCmd extends Matchers with ScalaFutures with SwaggerValidator {
     Try(resp.entity.toStrict(timeout).map { _.data }.map(_.utf8String).futureValue).getOrElse("")
   }
 
+  def getTransactionId(resp: HttpResponse): String = {
+    val tidHeader = resp.headers.find(_.is(TransactionId.generatorConfig.lowerCaseHeader))
+    withClue(
+      s"The header ${TransactionId.generatorConfig} is not set. This means that the request did not reach nginx (and not controller if nginx is skipped in that test).") {
+      tidHeader shouldBe defined
+    }
+    tidHeader.get.value()
```

Comment:
can drop `()'`?

Reply:
`and not controller if nginx is skipped in that test` -> `or the controller if nginx is skipped in that test`

Reply:
Thanks for the feedback. Both is addressed.